test/index.test.ts

```diff
@@ -192,6 +193,340 @@ describe('HipThrusTS', () => {
       });
     });
   });
```

Comment:
see comments from the other test...
also, don't forget to make sure to test the "bad" case using TS 3.9 ;)
- [ ] tests

Reply:
tests task done in this pr